.github/workflows/release.yml

```
@@ -0,0 +1,150 @@
+          # Package ARM64
+          cd target/aarch64-unknown-linux-gnu/release
+          tar -czf ../../../release-artifacts/rock-node-${{ github.event.release.tag_name }}-linux-arm64.tar.gz rock-node
+          cd -
+
+          # Generate checksums
+          cd release-artifacts
+          sha256sum * > checksums.txt
+          cd -
+
+      - name: Upload Release Assets
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)